prov/sockets/configure.m4

```diff
@@ -23,7 +31,7 @@ AC_DEFUN([FI_SOCKETS_CONFIGURE],[
 				[],
 				[],
 				[sockets_shm_happy=1],
-				[sockets_shm_happy=0])
+				[sockets_shm_happy=0])])
```

Comment:
Just curious -- how did the previous code work *at all* without the `])` that you just added?